### ISSUE TITLE
Possible Feature Request

### ISSUE THREAD

**Issue by Saereth** [MtcLeo05/BotaniaEditor#1]
Just wanted to start off by saying I really like what you're doing with the mod so far, well done :P

Now the idea, im not sure if this would even be possible but the idea would be to allow us to define different cakes as producing different quantities of mana for the kekimaru. So a vanilla cake produces x amount, a honey cake produces a different amount, chocolate cake from pams or something. Config wise I'd imagine it would just be a list of blockid:manavalue or maybe some other implementation entirely? If this isnt something doable thats fine but I figured I'd ask. Thanks again!

**Comment by MtcLuna05** [comment]
Thanks for the suggestion! Once I can I'll check if it's possible.

**Comment by MtcLuna05** [comment]
Just checked, kekimurus natively supports any block of the same kind of the vanilla cake and adds a constant mana. It technically could possible to add something like what you asked, but since some blocks add their own way to implement cakes, and don't use the vanilla way I would need to manually add each block and then make how much mana each cake gives.

TLDR: It's really hard to make it fully customizable, with my knowledge the max I could do is hardcode each cake, and then make only how much mana it's given customizable.

**Comment by Saereth** [comment]
I suspected it would be something like this tbh, that's understandable. Thanks for having a look :P
